src/middlewared/middlewared/plugins/service_/services/all.py

```diff
@@ -55,10 +55,10 @@
     SysctlService,
     SyslogdService,
     SystemService,
-    SystemDatasetsService,
     TimeservicesService,
     TtysService,
     UserService,
+    WinbinddService,
```

Comment:
Why are we adding a WinbinddService when we already have IdmapService that wraps around winbindd?